oak_ml_transparency/mnist/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Using the same base image that is used in https://github.com/Trusted-AI/adversarial-robustness-toolbox
```

Comment:
Is this a Dockerfile specifically for MNIST?
Or it could be reused by other models as well?

I.e. it's currently in the `mnist` folder

Reply:
This is for MNIST. The idea is that each evaluation has to provide its own Dockerfile. 

But the Dockerfile has to follow a specific format, specifying which is still WIP.